docs/reference/ml/apis/jobcounts.asciidoc

```diff
@@ -89,13 +89,13 @@ or old results are deleted, the job counts are not reset.
   (string) A unique identifier for the job.
 
 `last_data_time`::
-  (datetime) The timestamp at which data was last analyzed, according to server time.
+  (date) The timestamp at which data was last analyzed, according to server time.
 
 `latest_empty_bucket_timestamp`::
   (date) The timestamp of the last bucket that did not contain any data.
 
 `latest_record_timestamp`::
-  (date) The timestamp of the last processed record.
+  (date) The timestamp of the last input document.
```

Comment:
Might be worth editing this to `(date) The timestamp of the latest chronologically input document.` for consistency with the `earliest_record_timestamp` entry.

Reply:
Done, thanks!